src/privatesend.h

```diff
@@ -206,60 +244,28 @@ class CDarksendBroadcastTx
         READWRITE(sigTime);
     }
 
+    friend bool operator==(const CDarksendBroadcastTx& a, const CDarksendBroadcastTx& b)
+    {
+        return a.tx == b.tx;
+    }
+    friend bool operator!=(const CDarksendBroadcastTx& a, const CDarksendBroadcastTx& b)
+    {
+        return !(a == b);
+    }
+
+    explicit operator bool() const
+    {
+        return *this != CDarksendBroadcastTx() && this->tx != CTransaction();
```

Comment:
I believe ` && tthis->tx != CTransaction();` is unnecessery.
`*this != CDarksendBroadcastTx()` checked this already  -> newly created CDarksendBroadcastTx() has tx == CTransaction()